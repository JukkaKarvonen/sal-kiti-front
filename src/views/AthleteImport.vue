<template>
  <div class="import">
    <AthleteImport />
  </div>
</template>

<script>
/**
 * Result import view
 */
import AthleteImport from "@/components/AthleteImport.vue";

export default {
  name: "AthleteImportView",
  components: {
    AthleteImport
  },
  created() {
    document.title =
      this.$t("title.athlete_import") + " | " + this.$t("title.prefix");
  }
};
</script>
