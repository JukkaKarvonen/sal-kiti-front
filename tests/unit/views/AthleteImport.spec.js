import { shallowMount, createLocalVue } from "@vue/test-utils";
import AthleteImport from "@/views/AthleteImport.vue";

import BootstrapVue from "bootstrap-vue";

const localVue = createLocalVue();
localVue.use(BootstrapVue);

describe("AthleteImport.vue", () => {
  it("renders content", () => {
    const wrapper = shallowMount(AthleteImport, {
      localVue
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
