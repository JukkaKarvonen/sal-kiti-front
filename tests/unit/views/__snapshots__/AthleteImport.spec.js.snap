// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AthleteImport.vue renders content 1`] = `
<div
  class="import"
>
  <athleteimport-stub />
</div>
`;
